flow/connectors/mysql/cdc.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/PeerDB-io/peerdb/flow/otel_metrics"
 	"github.com/PeerDB-io/peerdb/flow/shared"
 	"github.com/PeerDB-io/peerdb/flow/shared/datatypes"
+	chmysql "github.com/PeerDB-io/peerdb/flow/shared/mysql"
```

Comment:
`chmysql` would be better as `qmysql` since this maintains abstraction re destination types